filters/auth/grantconfig.go

```diff
@@ -200,6 +201,10 @@ func (c *OAuthConfig) Init() error {
 
 	c.flowState = newFlowState(c.Secrets, c.SecretFile)
 
+	if c.ClientID == "" {
+		c.ClientID, _ = os.LookupEnv("OAUTH2_CLIENT_ID")
```

Comment:
I think it makes sense to use the bool for logging.

Reply:
Do you mean to log empty values for client id and secret? What should be the logging level?
The current version always fetches the env var for oidc so logging that might not make sense.

Reply:
> What should be the logging level?

log on debug level, that the fail back also failed to fetch the value.

Reply:
but it's a minor anyway.